workers/ohsome_quality_analyst/indicators/mapping_saturation/indicator.py

```diff
@@ -44,8 +44,6 @@ async def preprocess(self) -> bool:
         query_results = await ohsome_client.query(
             layer=self.layer, bpolys=json.dumps(self.bpolys), time=self.time_range
         )
-        if query_results is None:
-            return False
```

Comment:
if you remove this check, why are you still returning a bool value that only can be `True`?

Reply:
Since ohsome.client module will not return None anymore but instead raises exception, there is no need for checking the query results and return a boolean in all preprocess functions anymore.

This leads me to question the return values of calculation and create_figure functions (also boolean) as well.
I will open an issues to discuss this. But for now those functions are required to return a boolean because of there definition as abstract method in the base class.